### ISSUE TITLE
A producer of electric charge has incoherent behavior at high warp speed

### ISSUE THREAD

**Issue by Twisted6** [Kerbalism/Kerbalism#887]
I can't go above 1000x warp on any ship without unloading the vessel. It's basically made the game unplayable.

Kerbalism 3.19
Kerbalism FFT 0.2.0
Kerbalism System Heat 0.5.0

Modlist:
AtmosphereAutopilot (Fly-By-Wire) (AtmosphereAutopilot v1.6.1)
B9 Part Switch (B9PartSwitch v2.20.0)
BetterKerbol (BetterKerbol v1.0.1)
BetterTimeWarpContinued (BetterTimeWarpCont 2.3.13)Breaking Ground (BreakingGround-DLC 1.7.1)
Chatterer (Chatterer 0.9.99)
Chatterer Extended (ChattererExtended 0.6.2)
ClickThrough Blocker (ClickThroughBlocker 1:2.1.10.21)
Community Resource Pack (CommunityResourcePack v112.0.1)
Community Terrain Texture Pack (CommunityTerrainTexturePack 1:1.0.5)
Cryo Tanks (CryoTanks 1.6.5)
Cryo Tanks Core (CryoTanks-Core 1.6.5)
Cryogenic Engines (CryoEngines 1:2.0.6)
Cryogenic Engines - Near Future Aeronautics (CryoEngines-NFAero 2.0.6)
Deadly Reentry Continued (DeadlyReentry 1:v.7.9.0)
Deployable Engines Plugin (DeployableEngines 1.3.1)
Distant Object Enhancement /L (DistantObject v2.1.1.15)
Distant Object Enhancement /L default config (DistantObject-default v2.1.1.15)
Docking Port Sound FX (DockingPortSoundFX v2.1.12)
Dynamic Battery Storage (DynamicBatteryStorage 2:2.2.5.0)
Editor Extensions Redux (EditorExtensionsRedux 3.4.5)
Environmental Visual Enhancements Redux (EnvironmentalVisualEnhancements 3:1.11.7.1)
Far Future Technologies (FarFutureTechnologies 1.2.0)
Ferram Aerospace Research Continued (FerramAerospaceResearchContinued 3:0.16.1.2)
Hangar Extender (HangerExtenderExtended 3.6.0.2)
Harmony 2 (Harmony2 2.2.1.0)
Heat Control (HeatControl 0.6.1)
Hide Empty Tech Tree Nodes (HideEmptyTechNodes 1.3.2)
Janet's Minor Planets (JanetsPlanets 1.3.4)
Kabrams Sun Flare Orange High (KabramsSunFlaresPack-Orange-High 001)
Kerbal Actuators (KerbalActuators v1.8.5)
Kerbal Atomics (KerbalAtomics 1:1.3.3)
Kerbal Atomics - Other Mod Support (KerbalAtomics-NTRModSupport 1.3.3)
Kerbal Engineer Redux (KerbalEngineerRedux 1.1.9.0)
Kerbal Joint Reinforcement Continued (KerbalJointReinforcementContinued v3.7.3.0)
Kerbalism (Kerbalism 3.19)
Kerbalism FFT (Kerbalism-FarFutureTechnologies 0.2.0)
Kerbalism System Heat (Kerbalism-SystemHeat 0.5.0)
Kopernicus Planetary System Modifier (Kopernicus 2:release-1.12.1-198)
KSP Community Fixes (KSPCommunityFixes 1.34.1)
KSPBurst (KSPBurst v1.5.5.1)
Making History (MakingHistory-DLC 1.12.1)
MechJeb 2 (MechJeb2 2.14.3.0)
MechJeb and Engineer for all! (MechJebForAll 1.3.0.7)
Minor Planets Expansion (MinorPlanetsExpansion 1.0.8)
Mk2 Stockalike Expansion (Mk2Expansion 2:1.9.1.4)
Mk3 Stockalike Expansion (Mk3Expansion 1.6.1.4)
ModularFlightIntegrator (ModularFlightIntegrator 1.2.10.0)
Module Manager (ModuleManager 4.2.3)
Near Future Aeronautics (NearFutureAeronautics 2.1.1)
Near Future Electrical (NearFutureElectrical 1.2.3)
Near Future Electrical Core (NearFutureElectrical-Core 1.2.3)
Near Future Exploration (NearFutureExploration 1.1.2)
Near Future IVA Props (NearFutureProps 1:0.7.1)
Near Future Launch Vehicles (NearFutureLaunchVehicles 2.2.0)
Near Future Propulsion (NearFuturePropulsion 1.3.5)
Near Future Solar (NearFutureSolar 1.3.2)
Near Future Solar Core (NearFutureSolar-Core 1.3.2)
Near Future Spacecraft (NearFutureSpacecraft 1.4.3)
Other Worlds - Story and Planet Mod (OtherWorlds 1:1.0.4)
Outer Planets Mod (OuterPlanetsMod 2:2.2.10)
Outer Planets Mod - Parallax with Scatters (Parallax-OuterPlanetsMod v1.0.5)
Parallax (Parallax 2.0.6)
Parallax - Stock Planet Textures (Parallax-StockTextures 2.0.0)
Parallax - Stock Scatter Textures (Parallax-StockScatterTextures 2.0.1)
PlanetShine (PlanetShine 0.2.6.6)
PlanetShine - Default configuration (PlanetShine-Config-Default 0.2.6.6)
Pood's Calm Nebula Skybox (PoodsCalmNebulaSkybox v1.3.0)
ReStock (ReStock 1.4.3)
Restock Waterfall Expansion (RestockWaterfallExpansion 2.2.0 (PR))
ReStock+ (ReStockPlus 1.4.3)
Scatterer (Scatterer 3:v0.0838)
Scatterer Default Config (Scatterer-config 3:v0.0838)
Shabby (Shabby 0.3.0.0)
SIMPLEX Kerbalism (SIMPLEXKerbalism 2:3.15)
Space Dust (SpaceDust 0.4.4)
Spectra (Spectra v1.6.5)
Stockalike Station Parts Expansion Redux (StationPartsExpansionRedux 2.0.10)
System Heat (SystemHeat 0.6.0)
System Heat - Nuclear Engine Configuration (SystemHeat-FissionEngines 0.6.0)
System Heat - Nuclear Reactor Configuration (SystemHeat-FissionReactors 0.6.0)
System Heat - Resource Converter Configuration (SystemHeat-Converters 0.6.0)
System Heat - Resource Harvester Configuration (SystemHeat-Harvesters 0.6.0)
TETRIX TechTree (TETRIXTechTree 4:2.23)
TextureReplacer (TextureReplacer v4.5.3)
Toolbar Controller (ToolbarController 1:0.1.9.11)
TUFX (TUFX 1.0.7.1)
TweakScale Redistributable (TweakScale-Redist v2.4.7.5)
Vertex Mitchell-Netravali Filtered Heightmap (VertexMitchellNetravaliHeightMap 0.3)
Waterfall - Restock (WaterfallRestock 0.2.3)
Waterfall Core (Waterfall 0.9.0)

**Comment by Clayell** [comment]
From what I've heard, this is likely a problem with dynamic battery storage

**Comment by KerballOne** [comment]
> > > I ended up having to disable life support and fuel boil-off.
> > 
> > 
> > Have you tried playing with EnforceCoherency disabled in settings.cfg ??
> 
> That disabled the 1000x warp limit but didn't fix the underlying issue

Correct, however the underlying issue was already described and acknowledged by Got.  
Since it's a fundamental issue with other mods, this open issue should be closed as either duplicate or will not fix.

**Comment by Twisted6** [comment]
> > Near/Far Future collection
> 
> Probably Dynamic Battery Storage, as I stated before.

Which is a dependency of Near/Far Future.

**Comment by Aebestach** [comment]
This issue has been resolved in version 3.40. If you encounter this again, feel free to reopen this issue.
